docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/_index.md

```diff
@@ -286,7 +286,7 @@ Your Grafana Mimir cluster can now ingest metrics in production.
 
 ## Configure clients to write metrics to Mimir
 
-To configure each client to remote-write metrics to Mimir, refer to [Configure Prometheus to write to Grafana Mimir](https://grafana.com/docs/helm-charts/mimir-distributed/latest/get-started-helm-charts/gs-external-access/#configure-prometheus-to-write-to-grafana-mimir)
+To configure each client to remote-write metrics to Mimir, refer to [Configure Prometheus to write to Grafana Mimir](https://grafana.com/docs/helm-charts/mimir-distributed/<MIMIR_HELM_VERSION>/get-started-helm-charts/gs-external-access/#configure-prometheus-to-write-to-grafana-mimir)
```

Comment:
So you're absolute right that the variable doesn't exist. To define it, we use cascading front matter that is passed on to all child pages. For example, https://github.com/grafana/mimir/blob/1cd260c3e11211c359448fc9058eeb32df463a8f/docs/sources/helm-charts/mimir-distributed/_index.md?plain=1#L10-L13

There's another aspect to version substitution which is that the fallback value is the current version of the documentation set and not just an empty version. There's more information in https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution

Reply:
So in this case, since there's no overriding `MIMIR_HELM_VERSION` configuration, the substitution will use the version inferred from the page. This is "latest" for the latest version and the specific semantic version for old versions of the documentation.

Reply:
only if i had read the manual first 👀 thanks for explaining @jdbaldry 🙏 

the default behaviour looks smart enough for us to merge this PR

Reply:
Always happy to help! I'm always happy when I can refer to docs to support my answer. I should make sure that's always the case :D